content/hardware/04.pro/boards/portenta-x8/tutorials/01.user-manual/content.md

```diff
@@ -1,1440 +1,1363 @@
+| Manage network-based buses (e.g., Modbus TCP, etc.)     | Manage field buses (e.g., Modbus RTU, CANbus, etc.)            |
+| Access to peripherals without concurrent access control | Access to peripherals without concurrent access control        |
+
+In addition to the above features, here is an overview of the board's main components:
+
+* **External memory**: The board features an onboard 16 GB eMMC Flash memory and 2 GB Low Power DDR4 DRAM.
+
+* **Wireless connectivity**: The board supports 2.4 GHz Wi-Fi® (802.11 b/g/n) and Bluetooth® 5.1, provided by the Murata® 1DX module. This high-performance Wi-Fi® and Bluetooth® module allows the Portenta X8 to communicate wirelessly with other devices and systems.
+
+* **Ethernet connectivity**: The board features an onboard, high-performance 1 Gbps Ethernet transceiver accessible through its High-Density connectors.
+
+* **Secure element**: The board includes a ready-to-use secure element, the SE050C2 from NXP®, tailored for IoT devices and offering advanced security features. This allows Portenta X8 to achieve PSA certification from ARM®. For more information, click [here](https://www.psacertified.org/products/portenta-x8/).
+
+* **USB connectivity**: The board features a USB-C port for power and data, which is also accessible through the board's High-Density connectors. This port is linked to a MIPI to USB-C/DisplayPort converter, enabling video output through the USB-C connection.
+
+* **Power management**: The Portenta X8 includes a power management integrated circuit (PMIC) to meet the demands of always-connected IoT devices.
+
+### Pinout
+
+![Portenta X8 Pinout](assets/ABX00049-pinout.jpg)
+
+The full pinout is available and downloadable as PDF from the link below:
+* [Portenta X8 Pinout](https://docs.arduino.cc/static/019dd9ac3b08f48192dcb1291d37aab9/ABX00049-full-pinout.pdf)
+
+### Datasheet
+
+The full datasheet is available and downloadable as PDF from the link below:
+* [Portenta X8 Datasheet](https://docs.arduino.cc/resources/datasheets/ABX00049-datasheet.pdf)
+
+### Schematics
+
+The full schematics are available and downloadable as PDF from the link below:
+* [Portenta X8 Schematics](https://docs.arduino.cc/resources/schematics/ABX00049-schematics.pdf)
+
+### STEP Files
+
+The full _STEP_ files are available and downloadable from the link below:
+* [Portenta X8 STEP files](../../downloads/ABX00049-step.zip)
+
+### Features Overview
+
+Portenta X8 integrates two main programming experiences: the **Yocto Linux** and popular **Arduino** environments.
+
+![Portenta X8 Features Overview](assets/portenta-x8-functionality-overview.png "Portenta X8 Features Overview")
+
+To explore specific sections in more detail, please click on the links below that interest you:
+
+* [Linux Environment](#linux-environment)
+* [Arduino Environment](#arduino-environment)
+* [Arduino Cloud](#portenta-x8-with-arduino-cloud)
+
+### Portenta X8: Linux & Arduino Interoperability
```

Comment:
I don't understand why this section is here. I think it would be relevant to have a small chapter explaining which kind of Linux OS is running on the board (with docker containers support) and then maybe mention as well of the possibility of using Arduino Sketches on M4. Just a brief introduction and then link the correspondent chapter and Portenta X8 fundamentals

Reply:
Updated!